src/utils/rfqm_health_check.ts

```diff
@@ -65,6 +65,28 @@ export function transformResultToShortResponse(result: HealthCheckResult): RfqmH
     };
 }
 
+/**
+ * Creates issues related to the server/API not specific to the worker farm.
+ */
+export function getHttpIssues(isMaintainenceMode: boolean, registryBalance: BigNumber): HealthCheckIssue[] {
+    const issues = [];
+    if (isMaintainenceMode) {
+        issues.push({
+            status: HealthCheckStatus.Maintenance,
+            description: 'RFQM is set to maintainence mode via the 0x API configuration',
+        });
+    }
+    if (registryBalance.isLessThan(new BigNumber(BALANCE_FAILED_THRESHOLD).shiftedBy(ETH_DECIMALS))) {
```

Comment:
can `new BigNumber(BALANCE_FAILED_THRESHOLD).shiftedBy(ETH_DECIMALS)` be a constant?

Reply:
done